test/libsolidity/syntaxTests/tryCatch/data_location_error_message.sol

```diff
@@ -0,0 +1,11 @@
+contract Foo {
+    function test() public {
+        try this.f() {}
+        catch Error(string reason) {}
+    }
+
+    function f() public {
+    }
+}
+// ----
+// TypeError 6651: (88-101): Data location must be "memory" for parameter in function, but none was given.
```

Comment:
Can you double check we have a working semantic test for this? i.e. with `string memory` or `bytes memory`

Reply:
Yes, for example test/libsolidity/semanticTests/tryCatch/panic.sol